match/lib/match/runner.rb

```diff
@@ -32,7 +42,9 @@ def run(params)
         git_user_email: params[:git_user_email],
         clone_branch_directly: params[:clone_branch_directly],
         type: params[:type].to_s,
-        platform: params[:platform].to_s
+        platform: params[:platform].to_s,
+        google_cloud_bucket_name: params[:google_cloud_bucket_name].to_s,
+        google_cloud_keys_file: params[:google_cloud_keys_file].to_s
```

Comment:
Why not pass in params like `Storage.for_mode(params[:storage_mode], **params)`? 

With this method plugins can't provide their own storage mode that has custom parameters